e2e-tests/tfm/cypress/e2e/journeys/utilisation-reports/edit-payment.spec.js

```diff
@@ -77,176 +79,250 @@ context(`${PDC_TEAMS.PDC_RECONCILE} users can edit payments`, () => {
 
-    clearFormValues();
+      clearFormValues();
 
-    getFeeRecordCheckbox(feeRecord.id).check();
+      getFeeRecordCheckbox(feeRecord.id).check();
 
-    cy.getInputByLabelText('Amount received').type('nonsense');
-    cy.getInputByLabelText('Day').type('nonsense');
-    cy.getInputByLabelText('Month').type('nonsense');
-    cy.getInputByLabelText('Year').type('nonsense');
-    cy.getInputByLabelText('Payment reference (optional)').type('This is valid');
+      cy.getInputByLabelText('Amount received').type('nonsense');
+      cy.getInputByLabelText('Day').type('nonsense');
+      cy.getInputByLabelText('Month').type('nonsense');
+      cy.getInputByLabelText('Year').type('nonsense');
+      cy.getInputByLabelText('Payment reference (optional)').type('This is valid');
 
-    pages.utilisationReportEditPaymentPage.clickSaveChangesButton();
+      pages.utilisationReportEditPaymentPage.clickSaveChangesButton();
 
-    getFeeRecordCheckbox(feeRecord.id).should('be.checked');
-    getFeeRecordCheckbox(anotherFeeRecord.id).should('not.be.checked');
+      getFeeRecordCheckbox(feeRecord.id).should('be.checked');
+      getFeeRecordCheckbox(anotherFeeRecord.id).should('not.be.checked');
 
-    cy.get('a').should('contain', 'Enter a valid amount received');
-    cy.get('a').should('contain', 'The date payment received must be a real date');
+      cy.get('a').should('contain', 'Enter a valid amount received');
+      cy.get('a').should('contain', 'The date payment received must be a real date');
 
-    cy.get('form').should('contain', 'Enter a valid amount received');
-    cy.get('form').should('contain', 'The date payment received must be a real date');
+      cy.get('form').should('contain', 'Enter a valid amount received');
+      cy.get('form').should('contain', 'The date payment received must be a real date');
 
-    cy.getInputByLabelText('Amount received').should('have.value', 'nonsense');
-    cy.getInputByLabelText('Day').should('have.value', 'nonsense');
-    cy.getInputByLabelText('Month').should('have.value', 'nonsense');
-    cy.getInputByLabelText('Year').should('have.value', 'nonsense');
-    cy.getInputByLabelText('Payment reference (optional)').should('have.value', 'This is valid');
+      cy.getInputByLabelText('Amount received').should('have.value', 'nonsense');
+      cy.getInputByLabelText('Day').should('have.value', 'nonsense');
+      cy.getInputByLabelText('Month').should('have.value', 'nonsense');
+      cy.getInputByLabelText('Year').should('have.value', 'nonsense');
+      cy.getInputByLabelText('Payment reference (optional)').should('have.value', 'This is valid');
+    });
   });
 
-  it('should return to the premium payments table after the user clicks the save changes button', () => {
-    pages.utilisationReportPage.premiumPaymentsTab.clickPaymentLink(paymentId);
+  describe('when the user clicks the save changes button', () => {
+    it('should return to the premium payments table', () => {
```

Comment:
This test duplicates one below with a better name so I think this should be removed